src/components/CompareResults/SubtestsResults/SubtestsResultsMain.tsx

```diff
@@ -55,21 +56,34 @@ function SubtestsResultsMain({ view }: SubtestsResultsMainProps) {
     }),
     content: style({
       display: 'flex',
-      justifyContent: 'flex-end',
+      justifyContent: 'space-between',
       alignItems: 'center',
     }),
+    buttons: style({
+      display: 'flex',
+      justifyContent: 'flex-end',
+      $nest: {
+        '.MuiButtonBase-root:nth-child(2)': {
+          marginLeft: Spacing.Default,
+        },
```

Comment:
It's easier to use `gap: Spacing.Default` directly on the flex container for this.


(but you don't need it with the suggestion above)